http-server/src/main/java/io/micronaut/http/server/HttpServerConfiguration.java

```diff
@@ -539,7 +540,18 @@ public long getMaxFileSize() {
          */
         @Override
         public boolean isEnabled() {
-            return enabled;
+            return enabled == null ? DEFAULT_ENABLED : enabled;
+        }
+
+        /**
+         * The default multipart enabled setting is false for servlets, but effectively
+         * true for Netty. This method returns the value only if set by the user.
+         *
+         * @return The enabled setting.
+         */
+        @Internal
+        public Optional<Boolean> getEnabled() {
```

Comment:
I had to create this method because the default is false. We can't change the default for Netty so I needed to get access to the property without a default to know if the user explicitly set the value to false or not. I think the default should change to true for 3.0 then we can remove this method.